app/services/edge_cache/bust_podcast_episode.rb

```diff
@@ -0,0 +1,18 @@
+module EdgeCache
+  class BustPodcastEpisode < Bust
+    def self.call(podcast_episode, path, podcast_slug)
+      return unless podcast_episode && path && podcast_slug
+
+      podcast_episode.purge
+      podcast_episode.purge_all
+
+      begin
```

Comment:
If we take a look at the original code...

https://github.com/forem/forem/blob/eee63068dcde7e342a0d204666f3ba98c938ea2c/app/labor/cache_buster.rb#L140-L153

We are making duplicate actions (`bust(path)`, `podcast_episode.purge`, and `podcast_episode.purge_all`). Is this intentional to cover an edge case?

cc: @benhalpern

Reply:
We used to not have the `purge` and `purge_all` functionality mapping properly to the surrogate keys, so this very well could be completely unneeded duplicate or reminiscent to some case not 100% handled by either version. But I'm not sure which would be the case....